src/app/services/firestore.ts

```diff
@@ -28,7 +28,7 @@ export const courseConverter = {
       location: data.location,
       professors: data.professors,
       students: data.students,
-      tas: data.TAs,
+      tas: data.tas,
       onDuty: data.onDuty,
       tags: data.tags,
       // questions: data.questions,
```

Comment:
I think that we should keep the id (and in Course type) when retrieving from Firestore so our data is consistent on client and db.

Reply:
Similar note, shouldn't our Question type have a uid as well?

Reply:
I believe that's unnecessary.

```ts
const doc = await getDoc(<some Id>) ;
if (doc.exists()) {
   doc.id // accessible
}
```